src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurSAELineType.java

```diff
@@ -0,0 +1,15 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+/**
+ * Helper enum for categorizing various types of SAE lines.

```

Comment:
I don't think this comment is needed.

Reply:
Ok, I'll remove the comment or replace if with a better description that describes what this enum is currently used for.